packages/inquirer/lib/ui/prompt.js

```diff
@@ -16,9 +16,9 @@ class PromptUI extends Base {
     this.prompts = prompts;
   }
 
-  run(questions) {
+  run(questions, answers) {
     // Keep global reference to the answers
-    this.answers = {};
+    this.answers = answers || {};
```

Comment:
We'll want to clone the `answers` object to make sure we're not mutating outside values.

Reply:
Lodash? Or {...answers} is enough?